lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_manager.cpp

```diff
@@ -182,10 +183,11 @@ status_code_e S1apStateManager::read_ue_state_from_db() {
 }
 
 void S1apStateManager::create_s1ap_imsi_map() {
-  s1ap_imsi_map_ = (s1ap_imsi_map_t*)calloc(1, sizeof(s1ap_imsi_map_t));
+  s1ap_imsi_map_ = new s1ap_imsi_map_t();
 
-  s1ap_imsi_map_->mme_ue_id_imsi_htbl =
-      hashtable_uint64_ts_create(max_ues_, nullptr, nullptr);
```

Comment:
Can we get rid of the max_ues config after this change?

Reply:
Yes! planning to remove as part of porting hash table, state_ue_ht